### PR TITLE
assignment-1

### PR DESCRIPTION
**Pull request by nairvish9299**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
adding code for each assignment question

## What did you learn from the changes you have made?
how to use knn classifier to classify the data set

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
train_test_split to split the dataset

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
yes, understanding the split vector

## How were these changes tested?
by executing the code and validating the output

## A reference to a related issue in your repository (if applicable)
na

## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by PatelVishakh — APPROVED** [review]
Assignment 1: Complete. Great work!

Few comments:

Q1)III) The type of variable is categorical, this in a data science setting is asking whether the variable is continuous, integer, categorial. 

Q2)II) We did not standardize the response variable Class because it is categorical and standardization is not meaningful for non-numeric labels.

**Comment by nairvish9299** [comment]
> Assignment 1: Complete. Great work!
> 
> Few comments:
> 
> Q1)III) The type of variable is categorical, this in a data science setting is asking whether the variable is continuous, integer, categorial.
> 
> Q2)II) We did not standardize the response variable Class because it is categorical and standardization is not meaningful for non-numeric labels.

Thank you, @PatelVishakh, for your comments.
